AxonDeepSeg/apply_model.py

```diff
@@ -150,6 +183,11 @@ def apply_convnet(path_acquisitions, acquisitions_resolutions, path_model_folder
     ########### STEP 4: Reconstruction of the segmented patches into segmentations of acquisitions and
     # resampling to the original size
 
+    # Keras Segmentation
+    """
+    for i in range(len(prediction
```

Comment:
This comment appears to be incomplete?

Reply:
Maybe I forgot to remove comments in `apply_model.py`. I will remove them and make it clean.